client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -107,6 +108,16 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 	const generatedDesigns = allDesigns?.generated?.designs || [];
 	const staticDesigns = allDesigns?.static?.designs || [];
 
+	const { data: selectedDesignDetails, isLoading: isLoadingSelectedDesignDetails } =
```

Comment:
Hmm, is it possible to call this endpoint on the ThemePreview component instead? I think this is not the responsibility of this component.

Reply:
I thought about that and decided that it'd be better to keep the data source and component separated for now, in case it needs to be reused somewhere else like theme showcase. What do you think?

Reply:
OK, makes sense for now :)